test/load/locustfile-performance.py

```diff
@@ -10,66 +10,142 @@
 import grpc
 from iroha import Iroha, IrohaGrpc
 from iroha import IrohaCrypto as ic
-
+import iroha
 import common.writer
 
+import random
+import gevent
+
 
 HOSTNAME = os.environ['HOSTNAME']
 ADMIN_PRIVATE_KEY = 'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70'
 
+TXS = dict() # hash -> sent time
+COMMITTED = set()
```

Comment:
It should probably be replaced with integer, since it is only used as `len(COMMITTED)`. Same for `SENT` and `BLOCKS`